auth/src/commonMain/kotlin/com/liftric/auth/jwt/CognitoIdToken.kt

```diff
@@ -33,6 +33,9 @@ data class CognitoIdTokenClaims(
     override val cognitoUsername: String,
     override val exp: Long,
     override val eventId: String,
+    override val jti: String? = null,
+    @SerialName("origin_jti")
```

Comment:
Do you need that one? Same applies for the annotation above? Just because `cognitoUsername` does not need that because that one is handled in the `IdTokenSerializer`.